examples/bridge/host_main.c

```diff
@@ -32,6 +32,7 @@
 #include <net/if.h>
 #include <arpa/inet.h>
 #include <netinet/in.h>
+#include <nuttx/net/ip.h>
```

Comment:
host can't use nuttx specific heade file

Reply:
Yes. Missed that. Will fix

Reply:
Done

Reply:
The UDP example needs the same change